code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -1026,3 +1026,33 @@
 		M.drowsyness += 1 * REM * delta_time
 	return ..()
 
+
+/datum/reagent/consumable/moth_milk
+	name = "Moth Milk"
+	description = "Whoever thought that milking moths is a good idea was totally wrong. Is it even milk?"
```

Comment:
```suggestion
	description = "Whoever thought that milking moths was a good idea was totally wrong. Is this even milk?"
```

Reply:
ath went out of his way to review the moth fetish pr

Reply:
> ath went out of his way to review the moth fetish pr

https://www.youtube.com/watch?v=07wuZ3EWMAA